date.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@/bin
+libdir=@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@
+includedir=@CMAKE_INSTALL_PREFIX@/include/date
+
+Name: date
+Description: A date and time library based on the C++11/14/17 <chrono> header
+Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -ltz
+Cflags: -I${includedir}
```

Comment:
Certain compile definitions are changing libtz ABI and should be included in Cflags when the library build is enabled: `HAS_STRING_VIEW`, `USE_OS_TZDB` and probably `ONLY_C_LOCALE`.

Reply:
CMake snippet to extract transitive public defines from target, generate string with -Dflags and work around unsupported generator substitutions in configure file:
```cmake
# Cflags: -I${includedir} @TZ_COMPILE_DEFINITIONS@
set( TZ_COMPILE_DEFINITIONS "$<IF:$<TARGET_EXISTS:tz>,-D$<JOIN:$<TARGET_PROPERTY:tz,INTERFACE_COMPILE_DEFINITIONS>, -D>,>" )
configure_file( date.pc.in date.pc.cf @ONLY )
file( GENERATE OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/date.pc"
    INPUT "${CMAKE_CURRENT_BINARY_DIR}/date.pc.cf" )
```